changelog.d/20240709_111157_derek_auto_create_json_token_path.rst

```diff
@@ -0,0 +1,6 @@
+
+Fixed
+~~~~~
+
+- When a JSONTokenStorage is used, the containing directory will be automatically be
+  created if it doesn't exist instead of erroring (on Mac).
```

Comment:
Since it's not mac-specific, I don't think we should note the platform. Since I was suggesting an edit, I also went ahead and restored the pr link text from the template (<code>:pr:\`998\`</code> also works).

```suggestion
  created if it doesn't exist instead of erroring. (:pr:`NUMBER`)
```

Reply:
I'm not sure if it's mac exclusive or not. I only know it doesn't work on mac.